apps/website/src/pages/updates.tsx

```diff
@@ -59,12 +59,31 @@ const UpdatesPage: NextPage = () => {
         </div>
 
         <div className="w-full max-w-lg">
-          <Image
-            src={bellPeepo}
-            alt=""
-            className="w-full"
-            unoptimized // Next.js optimization makes this larger than source
-          />
+          {/* Video player shown if no reduced-motion or reduced-data preference */}
+          <video
+            className="video-reduced reduced-motion:hidden reduced-data:hidden"
+            width="100%"
+            height="100%"
+            autoPlay
+            loop
+            muted
+            playsInline
+          >
+            <source src="/assets/bell-peepo.webm" type="video/webm" />
```

Comment:
Why are we placing this in the public assets, rather than it being an asset import?

Reply:
I spent a bit trying to get the video loader to accept it but was having a bunch of trouble getting it working.

I can take another stab at it if that is preferable.

Reply:
Completely missed this sorry, don't run it through the video loader, just mark the file extension as a regular asset in the webpack config if it isn't already.